src/i18n/resources/fr.js

```diff
@@ -480,6 +480,22 @@ L'équipe de FRA
     invitationLink: "Lien d'invitation",
     sendInvitation: "Envoyer un email d'invitation",
     inviteAgain: 'Inviter de nouveau',
+    invitationAcceptedNotification: {
+      subject: 'L’utilisateur {{invitedUserName}} {{invitedUserSurname}} a accepté votre invitation',
+      textMessage: `Cher {{recipientName}} {{recipientSurname}},
+
+L’utilisateur {{invitedUserName}} {{invitedUserSurname}} a accepté votre invitation à participer à {{assessmentName}} {{cycleName}} en tant que {{role}} pour le/la {{country}}
+{{- manageCollaboratorsUrl}}
+
+L’équipe FRA
+    `,
+      htmlMessage: `Cher {{recipientName}} {{recipientSurname}},
+<br/><br/>
+L’utilisateur {{invitedUserName}} {{invitedUserSurname}} a accepté votre invitation à participer à {{assessmentName}} {{cycleName}} en tant que {{role}} pour le/la <b><a href="{{- manageCollaboratorsUrl}}">{{country}}</a></b>.
+<br/><br/>
+L’équipe FRA
+    `,
+    },
```

Comment:
In the received email for the translation, we have:
"**Cher [for a man]/Chère [for a women]** {{recipient name}} {{recipient surname}}, ..."
But based on the other translations we have, like in the [invitation email](https://github.com/openforis/fra-platform/pull/3515/commits/5faeef68a4f8356986c3f096fb7cf6d78328d3cc#diff-73afee1ef09cb9f4f1a71b6be625a7ea1726173308b8d1e2f0383daaaa766182L497), we are only using "Cher". Should I add any logic to handle this using, maybe, the appellation?